p4-16/spec/grammar.mdk

```diff
@@ -334,6 +334,7 @@ structField
 
 enumDeclaration
     : optAnnotations ENUM name '{' identifierList '}'
+    | optAnnotations ENUM typeRef name '{' specifiedIdentifierList '}'
```

Comment:
I have implemented this grammar change in the compiler and it will work out (using "bit" in fact) - there are no ambiguities.

Reply:
I've left this alone for now, if you prefer bit, let me know.